cataloging/src/settings.js

```diff
@@ -138,6 +138,9 @@ export default {
       },
     ],
   },
+  protectedProperties: [
```

Comment:
This could maybe also include chains `["publication", "date"]`, and (step 2) be a calculated list from category-tagged properties/chains in vocab?

Reply:
I will look into the chain/property path part right away. Seems reasonable!